services/fdb/client/client.go

```diff
@@ -3461,3 +3566,193 @@ func (p *fdbServerVersionCmd) Execute(ctx context.Context, f *flag.FlagSet, args
+
+// WithoutCancel returns a context that is never canceled.
+func WithoutCancel(ctx context.Context) context.Context {
+	return noCancel{ctx: ctx}
+}
+
+type fdbMoveDataCmd struct{}
+
+func (*fdbMoveDataCmd) Name() string             { return fdbMoveDataCLIPackage }
+func (*fdbMoveDataCmd) SetFlags(_ *flag.FlagSet) {}
+func (*fdbMoveDataCmd) Synopsis() string {
+	return "Copy data across two tenant groups in a metacluster.\n" + client.GenerateSynopsis(GetSubpackage(flag.NewFlagSet("", flag.ContinueOnError)), 4)
+}
+func (r *fdbMoveDataCmd) Usage() string {
+	return client.GenerateUsage(fdbMoveDataCLIPackage, r.Synopsis())
+}
+
+func (r *fdbMoveDataCmd) Execute(ctx context.Context, f *flag.FlagSet, args ...interface{}) subcommands.ExitStatus {
+	c := GetSubpackage(f)
+	return c.Execute(ctx, args...)
+}
+
+type fdbMoveDataCopyCmd struct {
+	req *pb.FDBMoveDataCopyRequest
+}
+
+func (*fdbMoveDataCopyCmd) Name() string { return "copy" }
+func (*fdbMoveDataCopyCmd) Synopsis() string {
+	return "Initiate data copy across two tenant groups in a metacluster. Starts a long-running command on the server."
+}
+func (r *fdbMoveDataCopyCmd) Usage() string {
+	return "fdbmovedata copy <clusterFile> <capacityGroupIdentifier> <sourceClusterName> <destinationClusterName> [numProcs]"
+}
+
+func (r *fdbMoveDataCopyCmd) SetFlags(f *flag.FlagSet) {
+	r.req = &pb.FDBMoveDataCopyRequest{}
+}
+
+func (r *fdbMoveDataCopyCmd) Execute(ctx context.Context, f *flag.FlagSet, args ...interface{}) subcommands.ExitStatus {
+	state := args[0].(*util.ExecuteState)
+	c := pb.NewFDBMoveClientProxy(state.Conn)
+
+	if f.NArg() < 4 || f.NArg() > 5 {
+		fmt.Fprintln(os.Stderr, "usage: ", r.Usage())
+		return subcommands.ExitFailure
+	}
+
+	clusterFile, tenantGroup, sourceCluster, destinationCluster := f.Arg(0), f.Arg(1), f.Arg(2), f.Arg(3)
+	// default number of processes is 10
+	numProcs := int64(10)
+	var s int64
+	var err error
+	if f.NArg() == 5 {
+		s, err = strconv.ParseInt(f.Arg(4), 10, 64)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "can't parse number of processes: %v\n", err)
+			return subcommands.ExitFailure
+		}
+		numProcs = s
+	}
+
+	resp, err := c.FDBMoveDataCopyOneMany(ctx, &pb.FDBMoveDataCopyRequest{
+		ClusterFile:        clusterFile,
+		TenantGroup:        tenantGroup,
+		SourceCluster:      sourceCluster,
+		DestinationCluster: destinationCluster,
+		NumProcs:           numProcs,
+	})
+	if err != nil {
+		// Emit this to every error file as it's not specific to a given target.
+		for _, e := range state.Err {
+			fmt.Fprintf(e, "fdb move data copy error: %v\n", err)
+		}
+
+		return subcommands.ExitFailure
+	}
+
+	retCode := subcommands.ExitSuccess
+	for r := range resp {
+		if r.Error != nil {
+			fmt.Fprintf(state.Err[r.Index], "fdb move data copy error: %v\n", r.Error)
+			retCode = subcommands.ExitFailure
+		}
+		fmt.Fprintf(state.Out[r.Index], "Command ID to wait on: %d\n", r.Resp.Id)
```

Comment:
What do you think about adding something like this?
`fmt.Fprintf("Run fdbmovedata wait %d to wait for the process to complete", r.Resp.Id)`